src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -442,9 +441,21 @@ def resize_subvolume(self, fs_handle, **kwargs):
                     raise VolumeException(
                         -errno.ENOENT, "Subvolume '{0}' not found, create it with " \
                         "'ceph fs subvolume create' before resizing subvolumes".format(subvolname))
-                nsize, usedbytes = sv.resize_subvolume(subvolpath, newsize, noshrink)
-                ret = 0, json.dumps([{'bytes_used': usedbytes},{'bytes_quota': nsize},
-                                     {'bytes_pcent': '{0:.2f}'.format((float(usedbytes) / nsize) * 100.0)}], indent=2), ""
+
+                try:
+                    noshrink = kwargs['no_shrink']
+                    nsize, usedbytes = sv.resize_subvolume(subvolpath, int(newsize), noshrink)
+                    ret = 0, json.dumps([{'bytes_used': usedbytes}, {'bytes_quota': nsize},
+                                         {'bytes_pcent': '{0:.2f}'.format((float(usedbytes) / nsize) * 100.0)}], indent=2), ""
+                except ValueError:
+                    newsize = newsize.lower()
+                    if newsize == "inf":
```

Comment:
"inf" or "infinite"

Reply:
It was changed from `infinite` to `inf` based on your suggestion https://github.com/ceph/ceph/pull/31157#discussion_r339169550.

Reply:
I mean you should accept both.

Reply:
@batrick Fixed.